plugins/plugin.xml

```diff
@@ -31,6 +31,14 @@
                 <param name="android-package" value="com.optimove.android.cordova.OptimoveSDKPlugin" />
             </feature>
             <preference name="AndroidXEnabled" value="true" />
+        </config-file >
+        <config-file parent="/manifest/application/activity" target="AndroidManifest.xml">
+            <intent-filter android:label="deepLabel" android:autoVerify="true">
+                 <action android:name="android.intent.action.VIEW" />
+                 <category android:name="android.intent.category.DEFAULT" />
+                 <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="https" android:host="cordova-example-optimove.lnk.click"/>
```

Comment:
is this part of SDK? But host must be per customer. What's the plan for this? Could
1) ask customers to add to manifest
2) also somehow read from some config and set when plugin is added

Reply:
yes you are right, maybe its best to be aligned with flutter and ask customers to add it to the manifest

Reply:
I added a section to the wiki with an explanation of what should be done [https://github.com/optimove-tech/Optimove-SDK-Cordova/wiki/deferred-deep-linking](url)

Reply:
not sure about Flutter, but cordova has many ways to tweak manifest etc. So,

1. For the manifest DDL configuration clients shall add DDL configuration to `config.xml`. This way done only once for app and they have full control of schemes / host / ...
2. ...